### ISSUE TITLE
Can't run session because not resized

### ISSUE THREAD

**Issue by alanMachineLeraning** [alibaba/MNN#1439]
Compute Shape Error for MobilenetV2/Conv/BatchNorm/FusedBatchNorm
Invalide module for quantized
Create exection error : 267
Can't run session because not resized
run.sh: line 1: 35669 Segmentation fault      python quant_aware_training.py --model_file 'quantization_aware_training_demo_resource/mobilenet_v2_tfpb_train_public.mnn' --val_image_path 'quantization_aware_training_demo_resource/ILSVRC2012_img_val/' --val_txt 'quantization_aware_training_demo_resource/val.txt'

**Comment by jxt1234** [comment]
If add --fortrain in convert, you need use https://www.yuque.com/mnn/cn/vg3to5#umBOX for quan

**Comment by violet17** [comment]
@jxt1234   hi, I met the same error when running benchmark.out using mobilenetv2
```
$ ~/projects/MNN/build/benchmark.out ./ 10 5 3 1
MNN benchmark
Forward type: **OpenCL** thread=1** precision=2
--------> Benchmarking... loop = 10, warmup = 5
Compute Shape Error for MobilenetV2/Conv/BatchNorm/FusedBatchNorm
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
Can't run session because not resized
[ - ] mobilenet_v2_1.0_224_frozen.mnn    max =    0.001 ms  min =    0.000 ms  avg =    0.000 ms
```

I convert mobilenetv2 pb file to mnn by using the following command:
```
$ ~/projects/MNN/build/MNNConvert -f TF --modelFile ../mobilenet_v2_1.0_224_frozen.pb  --MNNModel mobilenet_v2_1.0_224_frozen.mnn --bizCode biz

MNNConverter Version: 0.2.1.5git - MNN @ 2018


Start to Convert Other Model Format To MNN Model...
Start to Optimize the MNN Net...
inputTensors : [ input, ]
outputTensors: [ MobilenetV2/Predictions/Reshape_1, ]
Converted Done!
```
The pb files is from Tensorflow github.

What should I do to fix it?
